src/apps/loan-list/materials/utils/material-fetch-hoc.tsx

```diff
@@ -48,7 +49,11 @@ const fetchMaterial =
         }
       }, [isSuccessManifestation, data]);
 
-      if (!material) return null;
+      // in cases where the material is not found we return null, else we would load forever
+      if (data && data.manifestation === null) return null;
```

Comment:
I think this could be nice to put in a global helper:
eg.: `materialDataIsValid()`